packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import {
+  css,
+  autobind,
+  BaseComponent
+} from '../../Utilities';
+import { IButtonProps } from '../../Button';
+import { IOverflowSetProps } from './OverflowSet.Props';
+import { FocusZone, FocusZoneDirection } from '../../FocusZone';
+
+import * as stylesImport from './OverflowSet.scss';
+const styles: any = stylesImport;
+
+export class OverflowSet extends BaseComponent<IOverflowSetProps, null> {
+
+  public render() {
+    let {
+      items,
+      overflowItems,
+      onRenderOverflowButton
+    } = this.props;
+
+    const overflowButtonProps: IButtonProps = {
+      menuProps: { items: overflowItems }
+    }
+
+    return (
+      <FocusZone className={ css('ms-OverflowSet', styles.root) } direction={ FocusZoneDirection.horizontal } role='menubar' >
+        { items && this._onRenderItems(items) }
+        { overflowItems.length && onRenderOverflowButton(overflowButtonProps) }
```

Comment:
For the onrenderoverflowbutton? I guess it makes sense given your other comment regarding how overflow items are rendered.